### DISCUSSION TITLE
`Intl` in `globals.es2015`?

### DISCUSSION THREAD

**Discussion by aladdin-add** [eslint/eslint#18317] [Q&A] [answered]
I recently ran into a problem, and after looking it up I realized that it was due to `globals v15`  adding `globals.es2015.Intl: false`, but in eslint v9, it maintains its own globals:
https://github.com/eslint/eslint/blob/1fa66220ad130eeb69cfa0207d3896b7bb09c576/conf/globals.js#L2
and it was not enabled.

Should we also update the globals in eslint?

/ @nzakas @fisker

**Comment by fisker** [comment, marked as answer]
Explained in the original PR https://github.com/sindresorhus/globals/pull/213#issue-2142294092

I think it's correct that it's an es6 global.

**Comment by nzakas** [comment]
Sure, let's add it.

**Comment by fisker** [comment]
> I recently ran into a problem

Out of Curiosity, what kind of problem can it be ... It was in `node` and `browser` environments, unless you didn't use any of them?
